Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.cpp

```diff
@@ -104,6 +104,8 @@ GStreamerRegistryScanner::ElementFactories::ElementFactories(OptionSet<ElementFa
         rtpPayloaderFactories = gst_element_factory_list_get_elements(GST_ELEMENT_FACTORY_TYPE_PAYLOADER, GST_RANK_MARGINAL);
     if (types.contains(Type::RtpDepayloader))
         rtpDepayloaderFactories = gst_element_factory_list_get_elements(GST_ELEMENT_FACTORY_TYPE_DEPAYLOADER, GST_RANK_MARGINAL);
+    if (types.contains(Type::Decryptor))
```

Comment:
I wonder if we should flag all the decryptor appearances in the scanner for `ENABLE(ENCRYPTED_MEDIA)` . I mean not only this part but almost all chunks of this patch.

Reply:
When EME is disabled no decryptor will be registered and the factory list will be empty. I think we can live without ifdefs.